packages/website/src/containers/content/pages/OfficerOverviewPage.tsx

```diff
@@ -26,9 +68,56 @@ export const OfficerOverviewPage: React.FC<OfficerOverviewPageProps> = ({ page }
         ))}
       </Sectionbar>
 
-
       <div className="LokiContainer">
-        <h1>Officer page</h1>
+        <div className="Layout Layout--sidebar-left" css={topOverviewStyles}>
+          <div>
+            <OneImage
+            aspectRatio={AspectRatio.r1by1}
+            src={page.officerImage.resource}
+            alt=""
+            />
+          </div>
+          <div>
+            <h2>{page.role}</h2>
+            <h1>{page.firstName} {page.lastName}</h1>
+            <div className="type-body-copy">
+              {convert(page.roleDescription)}
+            </div>
+            {/* SOCIAL MEDIA SECTION */}
+            <div css={socialStyles}>
+              <h4>{'Follow ' + page.firstName }</h4>
+              <ul className={cx('Social')} css={socialItemsStyles}>
```

Comment:
cx is a function for conditionally use css classes, we can remove it here. Use just `className="Social"`